requirements/local.txt

```diff
@@ -182,7 +198,7 @@ iniconfig==2.0.0
     # via
     #   -r requirements/test.txt
     #   pytest
-isort==5.12.0
+isort==5.13.0
```

Comment:
I'm going to block this PR for now. isort 5.13.0 [pulls in many dev dependencies as runtime dependencies](https://github.com/PyCQA/isort/issues/2206), and we don't really want that. Let's wait until that version is blocked via a constraint, or the issue is fixed upstream in the next minor isort release. (Fix is already merged to isort master)